app/services/selection.js

```diff
@@ -109,42 +109,81 @@ export default Ember.Service.extend({
       }
     }
 
-    // sigh...
-    if (
-      (toLevel === 'pumas' && fromLevel === 'ntas')
-      || (toLevel === 'pumas' && fromLevel === 'tracts')
-      || (toLevel === 'ntas' && fromLevel === 'pumas')
-      || (toLevel === 'blocks' && fromLevel === 'ntas')
-      || (toLevel === 'blocks' && fromLevel === 'pumas')
-      || (toLevel === 'ntas' && fromLevel === 'blocks')
-      || (toLevel === 'pumas' && fromLevel === 'blocks')
-    ) {
-      this.clearSelection();
-      return;
-    }
-
     if (this.get('selectedCount')) {
+      // sigh...
```

Comment:
Why is this still necessary? It looks like we're checking for some known combination of origin/destinations... doesn't it work for all of them now?